examples/hello-world/src/main.rs

```diff
@@ -21,6 +22,13 @@ async fn main() -> Result<ExitCode, Error> {
     // Create a new application.
     let mut app = via::app(());
 
+    app.include(|request: Request, next: Next| async {
+        Ok(next.call(request).await?.map(|body| {
+            let scc = stderr();
+            body.tee(scc).boxed()
```

Comment:
Let us pray that the SCC doesn't have anyone's JWT! 🙏 

The fact that I'm using acronyms means that miracles do happen. So at least there is hope..

Reply:
Oh shit I was wrong about being able to use acronyms :scream:

Reply:
When the SEC steals a JWT in an audit log, nobody wins.

This isn't a commitment to help people commit financial crimes but rather an attempt to prevent them from getting off the hook for financial crimes.

That's why the multi threaded runtime was implemented first. I care deeply about due process. ✌️